src/Kernel/BTrees.wl

```diff
@@ -196,6 +288,19 @@ SetAttributes[BTreeGamma, Listable];
 BTreeSigma[(BTree | BTreeN | BTreeDAE)[t_, ___]] := treeSigma[t];
 SetAttributes[BTreeSigma, Listable];
 
+BTreePrune[h_[t_,p__],h_[t_,p__]]/;(AnyTrue[{BTree,BTreeN,BTreeDAE},h===#&]):=h[\[FormalY],p];
+BTreePrune[h_[t_,p__],h_[k_,p__]]/;(AnyTrue[{BTree,BTreeN,BTreeDAE},h===#&]):=h[Expand[prune[t,k]],p];
+
+BTreeContract[h_[t_,p__],h_[k_,p__]]/;(AnyTrue[{BTree,BTreeN,BTreeDAE},h===#&]):=h[Expand[contract[t,k]],p];
+
+BTreeSubTrees[h_[t_,p__]]/;(AnyTrue[{BTree,BTreeN,BTreeDAE},h===#&]):=Map[h[#,p]&,Complement[DeleteDuplicates[subTrees[t]], {t}]];
```

Comment:
I think it's more intuitive and conventional if this returned a list instead of a product.  I understand internally the product is probably simpler, but it's not too bad to switch: `Times @@ list`